common/desktop-exports

```diff
@@ -141,38 +143,59 @@ update_xdg_dirs_values
 
 # Setup user-dirs.* or run xdg-user-dirs-update if needed
 needs_xdg_update=false
+needs_xdg_reload=false
 needs_xdg_links=false
+
+if [ $HOME != $SNAP_USER_DATA ] && ! is_subpath $XDG_CONFIG_HOME $HOME; then
```

Comment:
Can you please enclose variables in double quotes? There are other places in your changes where this could be done, too.

Reply:
Right, good catch... Fixed.